le_utils/constants/format_presets.py

```diff
@@ -54,6 +54,15 @@
 HTML5_DEPENDENCY_ZIP = "html5_dependency"
 HTML5_DEPENDENCY_ZIP_READABLE = "HTML5 Dependency (Zip format)"
 
+SLIDE_IMAGE = "slide_image"
```

Comment:
@indirectlylit suggested changing the names `SLIDE_IMAGE` and `slide_image` to 
`SLIDESHOW_IMAGE` and `slideshow_image`

@nucleogenesis I let you see what makes more sense